src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -695,7 +839,7 @@ bool SequentialSfMReconstructionEngine::MakeInitialPair3D(const Pair & current_p
         pose_I, cam_I, pose_J, cam_J, ob_xI.x, ob_xJ.x);
       const Vec2 residual_I = cam_I->residual(pose_I, landmark.X, ob_xI.x);
       const Vec2 residual_J = cam_J->residual(pose_J, landmark.X, ob_xJ.x);
-      if ( angle > 2.0 &&
+      if ( angle > 3.0 &&
```

Comment:
maybe use a 
```c++
const double minAngle = 3.0;
```
so that it will be easier if one day we will manage to expose the parameters/